src/sass/lightbox-dialog/lightbox-dialog.scss

```diff
@@ -186,13 +173,6 @@ button.icon-btn.lightbox-dialog__prev {
     width: 24px;
 }
 
-// Deprecated, remove next major
-// This is tied with handle, once we remove handle this needs to be removed
-.lightbox-dialog__window--expanded {
-    height: 95%;
```

Comment:
Is this accompanied by a change so that the dialog is essentially _always_ in this expanded state? Even if we do that, I'm not convinced that every dialog should be almost fullscreen

Reply:
Agree. We need sensible defaults for our major breakpoints. Almost fullscreen would _not_ be a sensible default for large screens. Design system should have defaults those specified. But also we probably need the capacity for apps to fine tune those when needed (using component level CSS variables?)

Reply:
This still needs further discussion, we will discuss offline

Reply:
Actually, I was mistaken. Looks like what I was thinking of is already handled by the `large` modifier